packages/cursorless-vscode/src/keyboard/grammar/getAcceptableTokenTypes.ts

```diff
@@ -76,19 +74,8 @@ function getRootStates(state: nearley.State) {
  * @returns A partial argument for the command that the state represents
  */
 function computePartialArg<T extends keyof KeyboardCommandHandler>(
-  state: nearley.State,
-) {
-  const { argNames } = getMetadata<T>(state);
-  let currentState = state;
-  const partialArg: Partial<Record<keyof KeyboardCommandArgTypes[T], any>> = {};
-
-  while (currentState.dot > 0) {
-    const argName = argNames[currentState.dot - 1]!;
-    if (argName !== UNUSED) {
-      partialArg[argName] = currentState.right?.data;
-    }
-    currentState = currentState.left!;
-  }
-
-  return partialArg;
+  _state: nearley.State,
+): Partial<Record<T, any>> {
+  // FIXME: Fill this out
```

Comment:
add a bit of context for when/why this needs fixing (e.g. when we improve the gui for partially typed commands)

Reply:
It's actually already done in https://github.com/cursorless-dev/cursorless/pull/2172